build-tools/packages/build-cli/package.json

```diff
@@ -122,7 +122,6 @@
 		"@fluid-private/readme-command": "workspace:~",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/eslint-config-fluid": "^4.0.0",
-		"@microsoft/api-extractor": "^7.39.1",
```

Comment:
This was duplicated between `dependencies` and `devDependencies`. This entry was removed.